spec.yml

```diff
@@ -213,14 +213,14 @@ paths:
             maxItems: 5
             items:
               $ref: '#/definitions/UserSearched'
-  /users/{uuid}/:
```

Comment:
Lol at calling this a `uuid` when it's not even. Generic _and_ wrong :slot_machine: